scenario/elasticreload.go

```diff
@@ -184,3 +231,24 @@ func logReloadDuration(started, ended string, logEntry *logger.LogEntry) {
 	duration := endedTS.Sub(startedTS)
 	logEntry.LogInfo("ReloadDuration", fmt.Sprintf("%.fs", duration.Seconds())) // format to no decimals since timestamp is with entire seconds only
 }
+
+func checkStatusOngoing(sessionState *session.State, actionState *action.State, host, id string) (bool, error) {
```

Comment:
Hybrid spooler technology like I suggested! Very nice

Reply:
only for edge case of websocket going down during reload action. Tested and works.